sbndqm/dqmAnalysis/PMT/SBND/fcl/pmt_online_monitor_SBND.fcl

```diff
@@ -0,0 +1,102 @@
+#include "simple_channel_info.fcl"
+#include "redis_connection.fcl"
+#include "opticaldetectormodules.fcl"
+#include "pmt_stream_config_SBND.fcl"
+
+physics: {
+  
+  producers: {
+
+    daqPMT: {
+      module_type: DaqDecoderSBNDPMT
+      FragmentLabels: [ "daq:CAENV1730", "daq:ContainerCAENV1730" ]
+    }
+
+  }
+
+  analyzers: {
+
+   pmtAnalysis: {
+
+      module_type: CAENV1730StreamsSBND
+      
+
+      OpDetWaveformLabel: "daqPMT"
+
+      reco_man:      @local::standard_preco_manager
+      HitAlgoConfig: @local::sbnd_opreco_hit_slidingwindow
+      PedAlgoConfig: @local::sbnd_opreco_pedestal_rmsslider 
+
+
+      PMTMetricConfig: {
+
+        hostname: "sbnd-db.fnal.gov"
+    
+        groups: {
+            PMT: [ [0, 380] ]
+        }
+
+        streams: ["30s"]
+        
+        metrics: {
+        
+          rms: {
+            units: ADC
+            title: "PMT channel %(instance)s rms"
+            display_range: [0., 50.]
+          }
+
+          pedestal: {
+            units: ADC
+            title: "%(group)s channel %(instance)s Pedestal"
+            display_range: [7000, 9000]
+          }
+          
+          rate: {
+            untis: Hz 
+            title: "%(group)s channel %(instance)s single PMT rate" 
+            display_range: [ 100000, 300000 ]
+          }
+
+        }   
+      }
+    }
+  }
+
+
+  my_producer_modules: [ daqPMT ]
+  trigger_paths: [ my_producer_modules ]
+  
+  ana: [ pmtAnalysis ]
+  end_paths: [ ana ]
+
+}
+
+
+services: {
+
+  TFileService: { fileName: "waveformtest.root"}
+
+  RedisConnection: @local::redis_connection
+
+  InitMetricManager: {
+    connections: {
+      fast: @local::redis_metric_fast
+      slow: @local::redis_metric_slow
+      archive: @local::redis_metric_archive
+    }
+  }
+
+} 
+
+
+services.RedisConnection.host: "sbnd-db.fnal.gov"
+
+source:
+{
+  module_type: RootInput
+  fileNames: ["/data1/sbndaq_dl01_r001248_sr01_20190702T192250_22_dl2.root"]
+}
```

Comment:
I would omit this section entirely. It's ok for testing, but writing a "temporary" input file in a general configuration is not a great idea. In addition, if no `source` section is present _art_ will assume `RootInput` as source module every time an input file is specified, so there is not much difference anyway.
```suggestion
```